readme.md

```diff
@@ -261,6 +261,123 @@ And can even be provided in the URI form:
+- In the constructor of the `Map` class, using the `projection` option. For globe:
+```ts
+const map = new maptilersdk.Map({
+  container: "map",
+  projection: "globe", // Force a globe projection
+});
+```
+or for Mercator:
+```ts
+const map = new maptilersdk.Map({
+  container: "map",
+  projection: "mercator", // Force a mercator projection
+})
+```
+This will overwrite the `projection` property from the style (if any) and will persist it later if the map style was to change.
+
+- Use the built-in methods:
+```ts
+const animate = false;
+map.enableGlobeProjection(animate);
+// or
+map.enableMercatorProjection(animate);
+```
+
+The animated transition is enabled by default, but can be disabled by passing `false`, as in the example above. Similarly to the `projection` option in the constructor, this will overwrite the projection settings from style (if any) and persist it when the style is updated.
+
+The projection setter built in Maplibre GL JS is also usable:
+```ts
+map.setProjection({type: "mercator"});
+// or
+map.setProjection({type: "globe"});
+```
+but this will not automatically animate the transition and may cause rendering glitches.
+
+- Using the `MaptilerProjectionControl`. Not mounted by default, it can easily be added with a single option in the `Map` constructor:
+```ts
+const map = new maptilersdk.Map({
+  container: "map",
+  projectionControl: true,  // or the position such as "top-left", "top-right", 
+});                         // "bottom-right" or "bottom-left"
+```
+This dedicated control will show a globe icon <img src="images/screenshots/globe_icon.png" width="30px"/> to transition from Mercator to globe projection and will show a flat map icon <img src="images/screenshots/mercator_icon.png" width="30px"/> to transition from globe to Mercator projection. The chosen projection persist with future style changes.
+
+## Field of view (FOV)
+The internal camera has a default vertical field of view [*(wikipedia)*](https://en.wikipedia.org/wiki/Field_of_view) of a wide ~36.86 degrees. In globe mode, such a large *FOV* reduces the amount of the Earth that can be seen at once and exaggerates the central part, comparably to a fisheye lens. In many cases, a narrower *FOV* is preferable. Here is how to update if:
+
+```ts
+// Ajust de FOV, with values from 1 to 50
+map.transform.setFov(10);
+map.redraw();
+```
+> 📣 *__Note:__* with the Mercator projection, it is possible to set a FOV of `0`, which yields a true orthographic projection [*(wikipedia)*](https://en.wikipedia.org/wiki/Orthographic_projection), but the globe projection does not allow this.
+
+Here is a table of FOV comparison:
+| 01°    | 10° | 20° | 30° | 40° | 50° | 
+| :--------: | :-------: |:-------: |:-------: |:-------: |:-------: |
+| ![](images/screenshots/fov_1.jpeg) | ![](images/screenshots/fov_10.jpeg) | ![](images/screenshots/fov_20.jpeg) | ![](images/screenshots/fov_30.jpeg) | ![](images/screenshots/fov_40.jpeg) | ![](images/screenshots/fov_50.jpeg) | 
+
```

Comment:
I'm thinking whether this is not too specific and maybe does not need to be documented directly in the readme? (Especially since it adds several megabytes of data to the repo.)

Reply:
Also, there is now a public method to adjust the FoV nicely: https://github.com/maplibre/maplibre-gl-js/commit/d3f2bca197ce9b5191406a0860b9ed9e61d5daa6

Reply:
I've update the FOV function with the new built-in. As per removing this section entirely, I think now that's it's here and it's not going to be written anywhere else in probably at least the next few months, we better keep it.